library/src/morpheus-master/sass/modules/tool/authz/_authz.scss

```diff
@@ -1,18 +1,18 @@
 .checkGrid th:hover,.checkGrid td h4:hover {
```

Comment:
This is a minor permissions component redesign. It removes the striped table effect and refines the hover effects of the various elements to better indicate what will happen when you click.